### ISSUE TITLE
Encountered an improper argument MPC-HC (Movie Player Classic)

### ISSUE THREAD

**Issue by brbbbq** [Kosinkadink/ComfyUI-VideoHelperSuite#198]
I'm getting this error when trying to playback video files in MPC-HC:
![240419-143104-mpc-hc64](https://github.com/Kosinkadink/ComfyUI-VideoHelperSuite/assets/135537138/423864ae-f12d-4a5a-b826-ae3d4500f2d5)

I think it might have something to do with the metadata because it goes away when I remove properties for the file:
![240419-144510-Screenshot](https://github.com/Kosinkadink/ComfyUI-VideoHelperSuite/assets/135537138/043ef78c-5847-4aed-8f32-0da5d4c97e7d)

**Comment by VisualBoy** [comment]
me too, very annoying

**Comment by Kosinkadink** [comment]
You should be able to save the video without metadata if you set save_metadata to False in the Combine Video node

**Comment by VisualBoy** [comment]
Yes, I know, but I would keep my workflow reference to easy changes
history... I would be cool to save it only on png

Il Mer 24 Lug 2024, 18:50 Jedrzej Kosinski ***@***.***> ha
scritto:

**Comment by 4lt3r3go** [comment]
yeah issue is related to the dumb mediaplayer classic itself, wich is super dumb and they havent solved this simple issue still today in 2024 😒
however you can remove metadata to open it in mediaplayer if you really need to use mediaplayer classic but at this point i'm just uninstalling it cause i lost hopes they fix this.
https://exiftool.org/ use this, get the exe and modify the name of the exe like this:  `exiftool(-all= -overwrite_original).exe`

**Comment by 4lt3r3go** [comment]
and is now fixed. 
you welcome

https://github.com/clsid2/mpc-hc/issues/3213
